README.md

```diff
@@ -20,43 +12,40 @@ The Titanium CLI requires [Node.js 0.10.x](http://nodejs.org/dist/) or newer.
 
     [sudo] npm install -g titanium
 
+After install, Titanium CLI is executable as `ti`    
+
 ## Bleeding Edge
 
-You can download the latest and greatest *unstable* Titanium CLI code by
-running the following:
+You can download the latest and greatest *unstable* Titanium CLI code by running the following:
 
-    [sudo] npm install -g git://github.com/appcelerator/titanium.git
+    [sudo] npm install -g git://github.com/tidev/titanium.git
 
 ## Obtaining a Titanium SDK
 
-You will need to download a Titanium SDK 3.0 or newer:
+You will need to download a Titanium SDK:
 
     # stable release (recommended)
-    titanium sdk install --default
+    ti sdk install --default
 
-There are times when the latest stable release has an issue or lacks a feature
-in which case you may want to try an unstable Titanium SDK build. These builds
-are not recommended for production use, but should work for development.
+There are times when the latest stable release has an issue or lacks a feature in which case you may want to try an unstable Titanium SDK build. These builds are not recommended for production use, but should work for development.
 
     # unstable next release
-    titanium sdk install --branch 3_1_X --default
+    ti sdk install --branch 3_1_X --default
```

Comment:
will that work again or shall we just remove this for now @cb1kenobi 
same with `--branch master`